### DISCUSSION TITLE
Git and Github UI support in Zed

### DISCUSSION THREAD

**Discussion by AroraShreshth** [zed-industries/zed#9083] [Feature Requests]
I have been using zed for a while now and find the rendering to be super smooth. As I use it regularly switching from VSCode, a visual toolset for git on sidebar and an ability for extensions to interact with git and provide quick information to users seem more than necessary. (cue the git panel on toolbar on and git lense features)

Are they on roadmap? if yes is there a way to contribute in this direction.

**Comment by bennetbo** [comment]
Yes it is on the roadmap, i believe you are looking for #4367 and #8665

**Comment by Moshyfawn** [comment]
We've received a plethora of feature requests for the UI Git integration. You can find them in #8665.

You can read [the contribution guide](https://github.com/zed-industries/zed/blob/main/CONTRIBUTING.md) for how to contribute.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
